### ISSUE TITLE
Right-to-Left Support in Timeline

### ISSUE THREAD

**Issue by kmusaied** [almende/vis#112]
I am working on an Arabic (Right-to-Left) site and when I use visJs it renders the timeline from Left to Right. Is there anyway to change the timeline direction?

issue sample: 
http://jsfiddle.net/kmusaied/tmskn/

**Comment by AlexDM0** [comment]
Hi,

Over the last year a lot of feature requests have been made. We have just introduced our new website which has a list of the requested features. We have placed this request on that list. 

The list can be found here:
http://visjs.org/featureRequests.html

An explaination of the new system can be found here:
http://visjs.org/blog.html#New\ website\ for\ vis\.js\!

I would like to stress that this does not mean we abandon this request. Discussion here will continue if needed on this feature but we will close it to keep our Github issue page more of a bug-todo list.

Future feature requests will still be made here and then added to the website by us.

Regards,

Alex
